examples/ci_cd/gitlab-ci-cd/main.tf

```diff
@@ -61,29 +61,20 @@ data "aws_eks_cluster_auth" "cluster" {
 }
 
 locals {
-  tenant      = "aws001"  # AWS account name or unique id for tenant
-  environment = "preprod" # Environment area eg., preprod or prod
-  zone        = "qa"      # Environment with in one sub_tenant or business unit
-
-  kubernetes_version = "1.21"
-
-  vpc_cidr     = "10.2.0.0/16"
-  vpc_name     = join("-", [local.tenant, local.environment, local.zone, "vpc"])
-  cluster_name = join("-", [local.tenant, local.environment, local.zone, "eks"])
-
-  terraform_version = "Terraform v1.0.1"
+  vpc_name     = join("-", [var.tenant, var.environment, var.zone, "vpc"])
+  cluster_name = join("-", [var.tenant, var.environment, var.zone, "eks"])
 }
 
 module "aws_vpc" {
-  source  = "terraform-aws-modules/vpc/aws"
-  version = "v3.2.0"
+  source = "terraform-aws-modules/vpc/aws"
+  //version = "v3.2.0"
 
   name = local.vpc_name
-  cidr = local.vpc_cidr
+  cidr = var.vpc_cidr
   azs  = data.aws_availability_zones.available.names
 
-  public_subnets  = [for k, v in slice(data.aws_availability_zones.available.names, 0, 3) : cidrsubnet(local.vpc_cidr, 8, k)]
-  private_subnets = [for k, v in slice(data.aws_availability_zones.available.names, 0, 3) : cidrsubnet(local.vpc_cidr, 8, k + 10)]
+  public_subnets  = [for k, v in data.aws_availability_zones.available.names : cidrsubnet(var.vpc_cidr, 8, k)]
+  private_subnets = [for k, v in data.aws_availability_zones.available.names : cidrsubnet(var.vpc_cidr, 8, k + 10)]
```

Comment:
Can you add the slice back? I removed it erroneously: https://github.com/aws-samples/aws-eks-accelerator-for-terraform/pull/212#discussion_r790018967

Reply:
Done , Updated the file